source/includes/fundamentals/code-examples/Clothing.cs

```diff
@@ -0,0 +1,24 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace DataFormats.Poco;
+
+// start-model
+public class Clothing
+{
+    public ObjectId Id { get; set; }
+
+    [BsonElement("name")]
+    public string Name { get; set; }
+
+    [BsonElement("inStock")]
+    public bool InStock { get; set; }
+
+    [BsonElement("price")]
+    [BsonRepresentation(BsonType.Double)]
```

Comment:
Why don't we use `BsonType.Decimal128` here?

Reply:
I agree. I've always used Decimal for financial values. I'll make that change.